OpenRA.Mods.Common/Traits/Render/WithAttackAnimation.cs

```diff
@@ -47,13 +54,25 @@ public WithAttackAnimation(ActorInitializer init, WithAttackAnimationInfo info)
 			wsb = init.Self.Trait<WithSpriteBody>();
 		}
 
-		public void Attacking(Actor self, Target target, Armament a, Barrel barrel)
+		void INotifyAttack.Attacking(Actor self, Target target, Armament a, Barrel barrel)
+		{
+			if (info.DelayRelativeTo == AttackDelayType.Attack && !string.IsNullOrEmpty(info.AttackSequence))
+				self.World.AddFrameEndTask(w => w.Add(new DelayedAction(info.Delay, () =>
+					{
```

Comment:
Hm, are you sure? the brackets belong to the `DelayedAction`, not the `AddFrameEndTask`.